README.md

```diff
@@ -223,238 +260,369 @@ To enable this feature:
-- `CHROME_EXTENSION_ID` :(Optional, LEGACY, default value: 'null'): The Chrome AddOn extension ID for screen sharing. Note: the Chrome extension is no longer required for screensharing.
-- `ENABLE_ANNOTATIONS`: (Optional, default value: true) Whether to enable annotations in screen sharing. This is only meaningful if screen sharing is not disabled.
+To enable and configure screen sharing, edit options in the config/config.json file
+or set environment variables:
+
+* `Screensharing.enabled` (config.json) / `ENABLE_SCREENSHARING` (environment variable) -- Whether
+  to enable screen sharing. The default value is `false`.
+
+* `Screensharing.chromeExtensionId` (config.json) / `CHROME_EXTENSION_ID`
+  (environment variable) -- The Chrome add-on extension ID for screen sharing.
+  Note: an extension is no longer required for screen sharing in Chrome 72+ and Opera 59+.
+  The browser prompts the end user for access to the screen as it would for access to the camera.
+  The default value is `null`.
+
+* `Screensharing.annotations` (config.json) / `ENABLE_ANNOTATIONS` (environment variable) -- Whether
+  to enable annotations in screen sharing. With annotations enabled, the room toolbar includes
+  an *Annotate* button is This is only meaningful if screen sharing is enabled.  The default value
+  is `true`.
+
+To learn more about how screen sharing works in OpenTok, see the [guide on screen
+sharing](tokbox.com/developer/guides/screen-sharing/).
+
+**Config.json example:**
 
 ```json
-"Screensharing": {
-    "enabled": false,
-    "chromeExtensionId": null,
-    "annotations": {
-        "enabled": true
-    }
+{
+    "Screensharing": {
+        "enabled": true,
+        "chromeExtensionId": "cfhdojbkjhnklbpkdaibdccddilifddb",
+        "annotations": {
+            "enabled": true
+        }
+    } 
 }
 ```
+**Environment variable example:**
 
-To know more about how screen sharing works in OpenTok, see the [guide on screen
-sharing](tokbox.com/developer/guides/screen-sharing/).
+```sh
+export ENABLE_SCREENSHARING=true;
+export CHROME_EXTENSION_ID=cfhdojbkjhnklbpkdaibdccddilifddb;
+export ENABLE_ANNOTATIONS=true;
+```
+
+### Feedback
 
-#### Feedback
+The app lets the developer POST feedback data to an endpoint on your HTTP server. To enable
+and configure this, edit options in the config/config.json file or set environment variables:
 
-The app lets the developer POST feedback data to an endpoint on your HTTP server:
+* `Feedback.url` (config.json) / `FEEDBACK_URL` (environment variable) -- The URL to send a POST
+  request with feedback data. Leave this as an empty string or undefined to disable issue reporting.
+  The default value is ''.
 
-`FEEDBACK_URL`: The URL to send a POST request with feedback data. Leave this as an empty string or
-undefined to disable issue reporting.
+* `Feedback.reportIssueLevel` (config.json) / `REPORT_ISSUE_LEVEL` (environment variable) -- The audio
```

Comment:
This section likely needs to be cleaned up now that we implemented HotJar. I'm not sure what it should say.

Reply:
I added some info on Hotjar. WRT reportIssueLevel, it is unused by Hotjar. I'll make that clear in the README.